people_detector/verified_obj[] veri_objs
